src/control/server/mgmt_system_test.go

```diff
@@ -117,6 +119,69 @@ func TestServer_MgmtSvc_LeaderQuery(t *testing.T) {
 	}
 }
 
+func TestServer_MgmtSvc_ClusterEvent(t *testing.T) {
+	rasEvent := ras.NewRankFailEvent(0, 0, nil)
+	rasEventPB := new(mgmtpb.RASEvent)
+	if err := convert.Types(rasEvent, rasEventPB); err != nil {
+		t.Fatal(err)
+	}
+
+	for name, tc := range map[string]struct {
+		req     *mgmtpb.ClusterEventReq
+		expResp *mgmtpb.ClusterEventResp
+		expErr  error
+	}{
+		"nil request": {
+			expErr: errors.New("nil request"),
+		},
+		"nil event": {
+			req:    &mgmtpb.ClusterEventReq{},
+			expErr: errors.New("unexpected event type"),
+		},
+		"successful notification": {
+			req: &mgmtpb.ClusterEventReq{
+				Event: &mgmtpb.ClusterEventReq_Ras{
+					Ras: rasEventPB,
+				},
+			},
+			expResp: &mgmtpb.ClusterEventResp{},
```

Comment:
We have sequence numbers for these, right? Should try with a nonzero sequence number and make sure it returns the same in the response.